evaluation/metrics/metrics.py

```diff
@@ -20,6 +21,83 @@
 
 
 # eval_metrics, take predictions and targets as input
+def get_dataperf_accuracy(predictions: list, targets: list):
```

Comment:
Can you modify the function name so that it is clear that you are computing the micro f1 score?